src/components/CategoryFilter.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import categories from '../variables/Categories';
+
+categories.push('All');
+const CategoryFilter = ({ handleOptionChange }) => (
+  <form>
+    <label htmlFor="category-filter">
+      <select name="category-filter" id="category-filter" required onChange={handleOptionChange}>
+        { categories.map(category => (
```

Comment:
You could try this ``` [...['All'], ...categories].map( category => { ``` instead of pushing to the array.
or
 You could try this ```  ['All', ...categories].map( category => { ``` instead of pushing to the array.

Reply:
Thank you so much for your suggestions.